### DISCUSSION TITLE
Openwrt 22.03 Asterisk 18 compilation

### DISCUSSION THREAD

**Discussion by GektorUA** [IchthysMaranatha/asterisk-chan-quectel#35] [General]
I am trying compile module on OpenWrt 22.03 with Asterisk 18 (formally package now has name just "Asterisk" - without version number in name), change in config all 16->18 and trying also without any version number, but always has error in compilation process.

**Comment by IchthysMaranatha** [comment]
You can have a look at the chan_dongle makefile for reference
https://git.cooluc.com/openwrt-feeds/telephony/src/branch/openwrt-22.03/net/asterisk-chan-dongle/Makefile

You could try with --with-astversion=18 but everywhere else just asterisk as in the ref makefile
